File: translator-projects-2025-10-06/translator-python/app/test_data.py
import requests

BASE_URL = "http://localhost:8000"

# 1. Agregar palabras
print("Agregando palabras...")
data = [
    {"word": "gato", "translation": "cat"},
    {"word": "árbol", "translation": "tree"},
    {"word": "cielo", "translation": "sky"}
]
r = requests.post(f"{BASE_URL}/add_words", json=data)
print("Add:", r.status_code, r.json())

# 2. Traducir una palabra
r = requests.get(f"{BASE_URL}/translate", params={"word": "gato"})
print("Translate single:", r.json())

# 3. Traducir muchas palabras
words = ["gato", "perro", "cielo"]
r = requests.post(f"{BASE_URL}/translate_many", json=words)
print("Translate many:", r.json())

File: translator-projects-2025-10-06/translator-node/src/test_data.js
import fetch from "node-fetch";

const BASE_URL = "http://localhost:8000";

async function main() {
  console.log("Agregando palabras...");
  const addRes = await fetch(`${BASE_URL}/add_words`, {
    method: "POST",
    headers: { "Content-Type": "application/json" },
    body: JSON.stringify([
      { word: "sol", translation: "sun" },
      { word: "luna", translation: "moon" },
      { word: "estrella", translation: "star" }
    ])
  });
  console.log("Add:", await addRes.json());

  const single = await fetch(`${BASE_URL}/translate?word=sol`);
  console.log("Translate single:", await single.json());

  const many = await fetch(`${BASE_URL}/translate_many`, {
    method: "POST",
    headers: { "Content-Type": "application/json" },
    body: JSON.stringify(["sol", "luna", "estrella"])
  });
  console.log("Translate many:", await many.json());
}

main();
